linfa-trees/examples/decision_tree.rs

```diff
@@ -18,8 +18,7 @@ fn accuracy(
     true_positive / labels.len() as f64
 }
 
-// A routine K-means task: build a synthetic dataset, fit the algorithm on it
-// and save both training data and predictions to disk.
+
```

Comment:
Clippy disallows double blank lines, I believe.

Reply:
Okay, good to know--let me double check the clippy notes to see if I missed a warning

Reply:
After running clippy again, I didn't get any warnings, so might leave this and see if the CI gets angry